export const createPost = async (req, res) => {
    try {
        console.log("In the create post method")
    } catch (error) {
        console.log(error);
    }
}